2.0.0-alpha.6

matteobruni opened the pull request.

matteobruni committed:
fix: emitters uses the main loop, so they pause/play with the same particles logic

matteobruni committed:
chore(release): published new version

 - ng-particles@2.3.1
 - inferno-particles@1.3.1
 - jquery-particles@1.20.1
 - preact-particles@1.20.1
 - react-tsparticles@1.20.1
 - svelte-particles@1.3.1
 - particles.vue@2.3.1
 - particles.vue3@1.3.1
 - tsparticles-editor@1.20.1
 - tsparticles@1.20.1
 - ng-particles-demo@1.5.1
 - editor-demo@1.5.1
 - inferno-particles-demo@1.3.1
 - jquery-particles-demo@1.5.1
 - tsparticles-demo@1.5.1
 - preact-particles-demo@1.5.1
 - react-tsparticles-demo@1.5.1
 - svelte-demo@1.3.1
 - particles.vue-demo@1.5.1
 - particles.vue3-demo@1.3.1
 - tsparticles-preset-60fps@1.5.1
 - tsparticles-preset-background-mask@1.5.1
 - tsparticles-preset-basic@1.5.1
 - tsparticles-preset-big-circles@1.5.1
 - tsparticles-preset-bouncing@1.5.1
 - tsparticles-preset-bubbles@1.5.1
 - tsparticles-preset-fire@1.5.1
 - tsparticles-preset-font-awesome@1.5.1
 - tsparticles-preset-fountain@1.5.1
 - tsparticles-preset-snow@1.5.1
 - tsparticles-preset-stars@1.5.1
 - tsparticles-shape-bubble@1.5.1
 - tsparticles-shape-heart@1.5.1
 - tsparticles-shape-multiline-text@1.5.1
 - tsparticles-shape-spiral@1.5.1
 - cra-template-particles-typescript@1.3.1
 - cra-template-particles@1.3.1
 - tsparticles-website@1.20.1

matteobruni committed:
Merge pull request #1376 from matteobruni/v1

1.20.1

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
feat: added new external shape rounded-rect, closes #1220

matteobruni committed:
Merge pull request #1377 from matteobruni/v1

Rounded Rect shape

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: fixed custom shapes, aligned to new architecture

matteobruni committed:
build: fixed emitters build

matteobruni committed:
build: fixed jquery component build

matteobruni committed:
build: fixed spiral, bubble and rounded-rect shapes and their samples

dependabot[bot] committed:
build(deps): bump @angular/platform-browser-dynamic

Bumps [@angular/platform-browser-dynamic](https://github.com/angular/angular/tree/HEAD/packages/platform-browser-dynamic) from 11.2.3 to 11.2.5.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/11.2.5/packages/platform-browser-dynamic)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @rollup/plugin-replace from 2.3.4 to 2.4.1

Bumps [@rollup/plugin-replace](https://github.com/rollup/plugins) from 2.3.4 to 2.4.1.
- [Release notes](https://github.com/rollup/plugins/releases)
- [Commits](https://github.com/rollup/plugins/compare/replace-v2.3.4...replace-v2.4.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump ts-loader from 8.0.17 to 8.0.18

Bumps [ts-loader](https://github.com/TypeStrong/ts-loader) from 8.0.17 to 8.0.18.
- [Release notes](https://github.com/TypeStrong/ts-loader/releases)
- [Changelog](https://github.com/TypeStrong/ts-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/TypeStrong/ts-loader/compare/v8.0.17...v8.0.18)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump webpack-bundle-analyzer from 4.3.0 to 4.4.0

Bumps [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer) from 4.3.0 to 4.4.0.
- [Release notes](https://github.com/webpack-contrib/webpack-bundle-analyzer/releases)
- [Changelog](https://github.com/webpack-contrib/webpack-bundle-analyzer/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/webpack-bundle-analyzer/compare/v4.3.0...v4.4.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1379 from matteobruni/dependabot/npm_and_yarn/dev/angular/platform-browser-dynamic-11.2.5

build(deps): bump @angular/platform-browser-dynamic from 11.2.3 to 11.2.5

matteobruni committed:
Merge pull request #1380 from matteobruni/dependabot/npm_and_yarn/dev/rollup/plugin-replace-2.4.1

build(deps-dev): bump @rollup/plugin-replace from 2.3.4 to 2.4.1

matteobruni committed:
Merge pull request #1381 from matteobruni/dependabot/npm_and_yarn/dev/ts-loader-8.0.18

build(deps-dev): bump ts-loader from 8.0.17 to 8.0.18

matteobruni committed:
Merge pull request #1382 from matteobruni/dependabot/npm_and_yarn/dev/webpack-bundle-analyzer-4.4.0

build(deps-dev): bump webpack-bundle-analyzer from 4.3.0 to 4.4.0

matteobruni committed:
build: updates

matteobruni committed:
feat: speed decay

matteobruni committed:
feat: speed decay

matteobruni committed:
fix: fixed decay default value

matteobruni committed:
Merge branch 'v1' into dev

matteobruni committed:
build: fixed merge

ghost commented:
## DeepCode's analysis on [#4199a1](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/44b26de08c968c79796a61b8eaf1f06525fee35d/matteobruni/tsparticles/4199a1f2a9430ea3b2cc484125990034f62a772e/pr/_/%2F/code/?utm_source=gh_review&c=0&w=1&i=3&) found:

- :warning: **1** warning, :information_source: **3** minor issues. :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">Testing a collection size for < 0 will always be false. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/4199a1f2a9430ea3b2cc484125990034f62a772e/plugins/absorbers/src/AbsorberInstance.ts#L104">AbsorberInstance.ts:104</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/44b26de08c968c79796a61b8eaf1f06525fee35d/matteobruni/tsparticles/4199a1f2a9430ea3b2cc484125990034f62a772e/pr/_/%2Fplugins%2Fabsorbers%2Fsrc%2FAbsorberInstance.ts/javascript%2Fdc_interfile_project%2FCollectionArraySizeMatch/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
    <tr>
      <td width="77%">non-arrow functions are forbidden Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/4199a1f2a9430ea3b2cc484125990034f62a772e/shapes/heart/src/index.ts#L6">index.ts:6</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/4199a1f2a9430ea3b2cc484125990034f62a772e/shapes/spiral/src/index.ts#L12">index.ts:12</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/4199a1f2a9430ea3b2cc484125990034f62a772e/shapes/confetti/src/index.ts#L26">index.ts:26</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/44b26de08c968c79796a61b8eaf1f06525fee35d/matteobruni/tsparticles/4199a1f2a9430ea3b2cc484125990034f62a772e/pr/_/%2Fshapes%2Fheart%2Fsrc%2Findex.ts/javascript%2Ftslint%2Fonly-arrow-functions/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
    <tr>
      <td width="77%">'public' is implicit. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/4199a1f2a9430ea3b2cc484125990034f62a772e/shapes/multiline-text/src/index.ts#L60">index.ts:60</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/4199a1f2a9430ea3b2cc484125990034f62a772e/shapes/multiline-text/src/index.ts#L78">index.ts:78</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/4199a1f2a9430ea3b2cc484125990034f62a772e/plugins/emitters/src/EmitterInstance.ts#L31">EmitterInstance.ts:31</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/44b26de08c968c79796a61b8eaf1f06525fee35d/matteobruni/tsparticles/4199a1f2a9430ea3b2cc484125990034f62a772e/pr/_/%2Fshapes%2Fmultiline-text%2Fsrc%2Findex.ts/javascript%2Ftslint%2Fmember-access/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/44b26de08c968c79796a61b8eaf1f06525fee35d/matteobruni/tsparticles/4199a1f2a9430ea3b2cc484125990034f62a772e/pr/_/%2F/code/?utm_source=gh_review&c=0&w=1&i=3&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
build: fixed a previous merge

matteobruni committed:
build: fixed a previous merge

matteobruni committed:
feat: confetti shape, closes #605

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
Merge branch 'v1' into dev

dependabot[bot] committed:
build(deps-dev): bump sass-loader from 11.0.0 to 11.0.1

Bumps [sass-loader](https://github.com/webpack-contrib/sass-loader) from 11.0.0 to 11.0.1.
- [Release notes](https://github.com/webpack-contrib/sass-loader/releases)
- [Changelog](https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/sass-loader/compare/v11.0.0...v11.0.1)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @babel/preset-typescript from 7.12.13 to 7.13.0

Bumps [@babel/preset-typescript](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-typescript) from 7.12.13 to 7.13.0.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.13.0/packages/babel-preset-typescript)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @octokit/core from 3.3.0 to 3.3.1

Bumps [@octokit/core](https://github.com/octokit/core.js) from 3.3.0 to 3.3.1.
- [Release notes](https://github.com/octokit/core.js/releases)
- [Commits](https://github.com/octokit/core.js/compare/v3.3.0...v3.3.1)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1385 from matteobruni/dependabot/npm_and_yarn/dev/sass-loader-11.0.1

build(deps-dev): bump sass-loader from 11.0.0 to 11.0.1

matteobruni committed:
Merge pull request #1387 from matteobruni/dependabot/npm_and_yarn/dev/babel/preset-typescript-7.13.0

build(deps-dev): bump @babel/preset-typescript from 7.12.13 to 7.13.0

matteobruni committed:
Merge pull request #1388 from matteobruni/dependabot/npm_and_yarn/dev/octokit/core-3.3.1

build(deps-dev): bump @octokit/core from 3.3.0 to 3.3.1

matteobruni committed:
feat: emitters spawn color

matteobruni committed:
Merge branch 'v1' of https://github.com/matteobruni/tsparticles into v1

matteobruni committed:
feat: animation loop count for opacity and size

matteobruni committed:
Merge branch 'v1' into dev

matteobruni committed:
build: fix merge

matteobruni committed:
build: fix merge

matteobruni committed:
build: fix merge

matteobruni committed:
build: fix merge

dependabot[bot] committed:
build(deps-dev): bump @babel/plugin-proposal-decorators

Bumps [@babel/plugin-proposal-decorators](https://github.com/babel/babel/tree/HEAD/packages/babel-plugin-proposal-decorators) from 7.12.13 to 7.13.5.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.13.5/packages/babel-plugin-proposal-decorators)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump css-loader from 5.0.1 to 5.1.3

Bumps [css-loader](https://github.com/webpack-contrib/css-loader) from 5.0.1 to 5.1.3.
- [Release notes](https://github.com/webpack-contrib/css-loader/releases)
- [Changelog](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/css-loader/compare/v5.0.1...v5.1.3)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump uglify-js from 3.13.0 to 3.13.1

Bumps [uglify-js](https://github.com/mishoo/UglifyJS) from 3.13.0 to 3.13.1.
- [Release notes](https://github.com/mishoo/UglifyJS/releases)
- [Commits](https://github.com/mishoo/UglifyJS/compare/v3.13.0...v3.13.1)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1389 from matteobruni/dependabot/npm_and_yarn/dev/babel/plugin-proposal-decorators-7.13.5

build(deps-dev): bump @babel/plugin-proposal-decorators from 7.12.13 to 7.13.5

matteobruni committed:
Merge pull request #1390 from matteobruni/dependabot/npm_and_yarn/dev/css-loader-5.1.3

build(deps-dev): bump css-loader from 5.0.1 to 5.1.3

matteobruni committed:
Merge pull request #1391 from matteobruni/dependabot/npm_and_yarn/dev/uglify-js-3.13.1

build(deps-dev): bump uglify-js from 3.13.0 to 3.13.1

matteobruni committed:
chore(release): published new version

 - ng-particles@2.4.0
 - inferno-particles@1.4.0
 - jquery-particles@1.21.0
 - preact-particles@1.21.0
 - react-tsparticles@1.21.0
 - svelte-particles@1.4.0
 - particles.vue@2.4.0
 - particles.vue3@1.4.0
 - tsparticles-editor@1.21.0
 - tsparticles@1.21.0
 - ng-particles-demo@1.6.0
 - editor-demo@1.6.0
 - inferno-particles-demo@1.4.0
 - jquery-particles-demo@1.6.0
 - tsparticles-demo@1.6.0
 - preact-particles-demo@1.6.0
 - react-tsparticles-demo@1.6.0
 - svelte-demo@1.4.0
 - particles.vue-demo@1.6.0
 - particles.vue3-demo@1.4.0
 - tsparticles-preset-60fps@1.6.0
 - tsparticles-preset-background-mask@1.6.0
 - tsparticles-preset-basic@1.6.0
 - tsparticles-preset-big-circles@1.6.0
 - tsparticles-preset-bouncing@1.6.0
 - tsparticles-preset-bubbles@1.6.0
 - tsparticles-preset-fire@1.6.0
 - tsparticles-preset-font-awesome@1.6.0
 - tsparticles-preset-fountain@1.6.0
 - tsparticles-preset-snow@1.6.0
 - tsparticles-preset-stars@1.6.0
 - tsparticles-shape-bubble@1.6.0
 - tsparticles-shape-confetti@1.6.0
 - tsparticles-shape-heart@1.6.0
 - tsparticles-shape-multiline-text@1.6.0
 - tsparticles-shape-rounded-rect@1.6.0
 - tsparticles-shape-spiral@1.6.0
 - cra-template-particles-typescript@1.4.0
 - cra-template-particles@1.4.0
 - tsparticles-website@1.21.0

matteobruni committed:
Merge pull request #1383 from matteobruni/v1

1.21.0

matteobruni committed:
build: schema

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
feat: added noise/path generator in plugins management

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
feat: ported confetti to v2 architecture

matteobruni committed:
refactor: renamed noise to path in move options, kept compatibility

matteobruni committed:
build: fixed noise/path options and editor

matteobruni committed:
build: updated old noise usage

matteobruni committed:
chore(release): published new version

 - ng-particles@2.5.0
 - inferno-particles@1.5.0
 - jquery-particles@1.22.0
 - preact-particles@1.22.0
 - react-tsparticles@1.22.0
 - svelte-particles@1.5.0
 - particles.vue@2.5.0
 - particles.vue3@1.5.0
 - tsparticles-editor@1.22.0
 - tsparticles@1.22.0
 - ng-particles-demo@1.7.0
 - editor-demo@1.7.0
 - inferno-particles-demo@1.5.0
 - jquery-particles-demo@1.7.0
 - tsparticles-demo@1.7.0
 - preact-particles-demo@1.7.0
 - react-tsparticles-demo@1.7.0
 - svelte-demo@1.5.0
 - particles.vue-demo@1.7.0
 - particles.vue3-demo@1.5.0
 - tsparticles-preset-60fps@1.7.0
 - tsparticles-preset-background-mask@1.7.0
 - tsparticles-preset-basic@1.7.0
 - tsparticles-preset-big-circles@1.7.0
 - tsparticles-preset-bouncing@1.7.0
 - tsparticles-preset-bubbles@1.7.0
 - tsparticles-preset-fire@1.7.0
 - tsparticles-preset-font-awesome@1.7.0
 - tsparticles-preset-fountain@1.7.0
 - tsparticles-preset-sea-anemone@1.7.0
 - tsparticles-preset-snow@1.7.0
 - tsparticles-preset-stars@1.7.0
 - tsparticles-shape-bubble@1.7.0
 - tsparticles-shape-confetti@1.7.0
 - tsparticles-shape-heart@1.7.0
 - tsparticles-shape-multiline-text@1.7.0
 - tsparticles-shape-rounded-rect@1.7.0
 - tsparticles-shape-spiral@1.7.0
 - cra-template-particles-typescript@1.5.0
 - cra-template-particles@1.5.0
 - tsparticles-website@1.22.0

matteobruni committed:
Merge pull request #1392 from matteobruni/v1

1.22.0

matteobruni committed:
build: website updates

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: fixed merge

matteobruni committed:
Merge pull request #1395 from matteobruni/v1

build: website updates

matteobruni committed:
Merge branch 'main' into dev

dependabot[bot] committed:
build(deps): bump preact from 10.5.12 to 10.5.13

Bumps [preact](https://github.com/preactjs/preact) from 10.5.12 to 10.5.13.
- [Release notes](https://github.com/preactjs/preact/releases)
- [Commits](https://github.com/preactjs/preact/compare/10.5.12...10.5.13)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @rollup/plugin-node-resolve from 11.1.1 to 11.2.0

Bumps [@rollup/plugin-node-resolve](https://github.com/rollup/plugins) from 11.1.1 to 11.2.0.
- [Release notes](https://github.com/rollup/plugins/releases)
- [Commits](https://github.com/rollup/plugins/compare/node-resolve-v11.1.1...node-resolve-v11.2.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1397 from matteobruni/dependabot/npm_and_yarn/dev/preact-10.5.13

build(deps): bump preact from 10.5.12 to 10.5.13

matteobruni committed:
Merge pull request #1398 from matteobruni/dependabot/npm_and_yarn/dev/rollup/plugin-node-resolve-11.2.0

build(deps-dev): bump @rollup/plugin-node-resolve from 11.1.1 to 11.2.0

matteobruni committed:
build: fixed size and opacity animations

matteobruni committed:
Merge branch 'v1' of https://github.com/matteobruni/tsparticles into v1

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
chore(release): published new version

 - ng-particles@2.5.1
 - inferno-particles@1.5.1
 - jquery-particles@1.22.1
 - preact-particles@1.22.1
 - react-tsparticles@1.22.1
 - svelte-particles@1.5.1
 - particles.vue@2.5.1
 - particles.vue3@1.5.1
 - tsparticles-editor@1.22.1
 - tsparticles@1.22.1
 - ng-particles-demo@1.7.1
 - editor-demo@1.7.1
 - inferno-particles-demo@1.5.1
 - jquery-particles-demo@1.7.1
 - tsparticles-demo@1.7.1
 - preact-particles-demo@1.7.1
 - react-tsparticles-demo@1.7.1
 - svelte-demo@1.5.1
 - particles.vue-demo@1.7.1
 - particles.vue3-demo@1.5.1
 - tsparticles-preset-60fps@1.7.1
 - tsparticles-preset-background-mask@1.7.1
 - tsparticles-preset-basic@1.7.1
 - tsparticles-preset-big-circles@1.7.1
 - tsparticles-preset-bouncing@1.7.1
 - tsparticles-preset-bubbles@1.7.1
 - tsparticles-preset-fire@1.7.1
 - tsparticles-preset-font-awesome@1.7.1
 - tsparticles-preset-fountain@1.7.1
 - tsparticles-preset-sea-anemone@1.7.1
 - tsparticles-preset-snow@1.7.1
 - tsparticles-preset-stars@1.7.1
 - tsparticles-shape-bubble@1.7.1
 - tsparticles-shape-confetti@1.7.1
 - tsparticles-shape-heart@1.7.1
 - tsparticles-shape-multiline-text@1.7.1
 - tsparticles-shape-rounded-rect@1.7.1
 - tsparticles-shape-spiral@1.7.1
 - cra-template-particles-typescript@1.5.1
 - cra-template-particles@1.5.1
 - tsparticles-website@1.22.1

matteobruni committed:
Merge pull request #1399 from matteobruni/v1

1.22.1

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: angular update

matteobruni committed:
build: updated inferno demo deps

matteobruni committed:
build: added confetti shape to vanilla demos

matteobruni committed:
build: tests updated

matteobruni committed:
feat: added Vector class taken from v2, this will be used wisely

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
feat: first round of vector on particles done

matteobruni committed:
build: website updated

matteobruni committed:
build: improved confetti shape

matteobruni committed:
build: particle position is now a vector

matteobruni committed:
build: fixed tests

matteobruni committed:
build: fixed some imports

matteobruni committed:
build: some code refactoring

matteobruni committed:
build: fixed tests

matteobruni committed:
build: improved absorbers

matteobruni committed:
build: used Vector in repulser

matteobruni committed:
build: prettiefied code

matteobruni committed:
build: improved particle velocity calc

matteobruni committed:
build: dependabot security updates

matteobruni committed:
build: improved confetti shape

matteobruni committed:
build: fixed import

matteobruni committed:
build: prettiefied code

matteobruni committed:
build: fixed some imports

matteobruni committed:
build: options now can accept any value, use at own risk

matteobruni committed:
chore(release): published new version

 - ng-particles@2.6.0
 - inferno-particles@1.6.0
 - jquery-particles@1.23.0
 - preact-particles@1.23.0
 - react-tsparticles@1.23.0
 - svelte-particles@1.6.0
 - particles.vue@2.6.0
 - particles.vue3@1.6.0
 - tsparticles-editor@1.23.0
 - tsparticles@1.23.0
 - ng-particles-demo@1.8.0
 - editor-demo@1.8.0
 - inferno-particles-demo@1.6.0
 - jquery-particles-demo@1.8.0
 - tsparticles-demo@1.8.0
 - preact-particles-demo@1.8.0
 - react-tsparticles-demo@1.8.0
 - svelte-demo@1.6.0
 - particles.vue-demo@1.8.0
 - particles.vue3-demo@1.6.0
 - tsparticles-preset-60fps@1.8.0
 - tsparticles-preset-background-mask@1.8.0
 - tsparticles-preset-basic@1.8.0
 - tsparticles-preset-big-circles@1.8.0
 - tsparticles-preset-bouncing@1.8.0
 - tsparticles-preset-bubbles@1.8.0
 - tsparticles-preset-fire@1.8.0
 - tsparticles-preset-font-awesome@1.8.0
 - tsparticles-preset-fountain@1.8.0
 - tsparticles-preset-sea-anemone@1.8.0
 - tsparticles-preset-snow@1.8.0
 - tsparticles-preset-stars@1.8.0
 - tsparticles-shape-bubble@1.8.0
 - tsparticles-shape-confetti@1.8.0
 - tsparticles-shape-heart@1.8.0
 - tsparticles-shape-multiline-text@1.8.0
 - tsparticles-shape-rounded-rect@1.8.0
 - tsparticles-shape-spiral@1.8.0
 - cra-template-particles-typescript@1.6.0
 - cra-template-particles@1.6.0
 - tsparticles-website@1.23.0

matteobruni committed:
Merge pull request #1401 from matteobruni/v1

1.23.0

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: updated to 1.23.0

dependabot[bot] committed:
build(deps-dev): bump mocha from 8.3.0 to 8.3.2

Bumps [mocha](https://github.com/mochajs/mocha) from 8.3.0 to 8.3.2.
- [Release notes](https://github.com/mochajs/mocha/releases)
- [Changelog](https://github.com/mochajs/mocha/blob/master/CHANGELOG.md)
- [Commits](https://github.com/mochajs/mocha/compare/v8.3.0...v8.3.2)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump rollup-plugin-typescript2 from 0.29.0 to 0.30.0

Bumps [rollup-plugin-typescript2](https://github.com/ezolenko/rollup-plugin-typescript2) from 0.29.0 to 0.30.0.
- [Release notes](https://github.com/ezolenko/rollup-plugin-typescript2/releases)
- [Commits](https://github.com/ezolenko/rollup-plugin-typescript2/compare/0.29.0...0.30.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump uglify-js from 3.13.1 to 3.13.2

Bumps [uglify-js](https://github.com/mishoo/UglifyJS) from 3.13.1 to 3.13.2.
- [Release notes](https://github.com/mishoo/UglifyJS/releases)
- [Commits](https://github.com/mishoo/UglifyJS/compare/v3.13.1...v3.13.2)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump core-js from 3.8.3 to 3.9.1

Bumps [core-js](https://github.com/zloirock/core-js/tree/HEAD/packages/core-js) from 3.8.3 to 3.9.1.
- [Release notes](https://github.com/zloirock/core-js/releases)
- [Changelog](https://github.com/zloirock/core-js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/zloirock/core-js/commits/v3.9.1/packages/core-js)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1402 from matteobruni/dependabot/npm_and_yarn/dev/mocha-8.3.2

build(deps-dev): bump mocha from 8.3.0 to 8.3.2

matteobruni committed:
Merge pull request #1403 from matteobruni/dependabot/npm_and_yarn/dev/rollup-plugin-typescript2-0.30.0

build(deps-dev): bump rollup-plugin-typescript2 from 0.29.0 to 0.30.0

matteobruni committed:
Merge pull request #1404 from matteobruni/dependabot/npm_and_yarn/dev/uglify-js-3.13.2

build(deps-dev): bump uglify-js from 3.13.1 to 3.13.2

matteobruni committed:
Merge pull request #1405 from matteobruni/dependabot/npm_and_yarn/dev/core-js-3.9.1

build(deps): bump core-js from 3.8.3 to 3.9.1

matteobruni committed:
build: website updated

matteobruni committed:
build: fixed website homepage configuration

matteobruni committed:
build: fixed some samples config

matteobruni committed:
Merge pull request #1406 from matteobruni/v1

build: website updated

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: aligned to v1.23.0

matteobruni committed:
build: updates

matteobruni committed:
fix: emitter play/pause were not working fine from container
feat: added autoPlay option to emitters

matteobruni committed:
fix: confetti fixed oval rotation

dependabot[bot] committed:
build(deps): bump preact-render-to-string from 5.1.14 to 5.1.16

Bumps [preact-render-to-string](https://github.com/developit/preact-render-to-string) from 5.1.14 to 5.1.16.
- [Release notes](https://github.com/developit/preact-render-to-string/releases)
- [Commits](https://github.com/developit/preact-render-to-string/compare/v5.1.14...v5.1.16)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump svelte-check from 1.2.4 to 1.2.6

Bumps [svelte-check](https://github.com/sveltejs/language-tools) from 1.2.4 to 1.2.6.
- [Release notes](https://github.com/sveltejs/language-tools/releases)
- [Commits](https://github.com/sveltejs/language-tools/compare/svelte-check-1.2.4...svelte-check-1.2.6)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1410 from matteobruni/dependabot/npm_and_yarn/dev/svelte-check-1.2.6

build(deps-dev): bump svelte-check from 1.2.4 to 1.2.6

matteobruni committed:
Merge pull request #1408 from matteobruni/dependabot/npm_and_yarn/dev/preact-render-to-string-5.1.16

build(deps): bump preact-render-to-string from 5.1.14 to 5.1.16

matteobruni committed:
build: updated install script

matteobruni committed:
feat: added drift (numeric or min/max object)
feat: added min/max alternative object to move.speed

matteobruni committed:
build: changed default confetti type to both

matteobruni committed:
feat: size, opacity and rotate particles options support now also min/max objects in value

matteobruni committed:
chore(release): published new version

 - ng-particles@2.7.0
 - inferno-particles@1.7.0
 - jquery-particles@1.24.0
 - preact-particles@1.24.0
 - react-tsparticles@1.24.0
 - svelte-particles@1.7.0
 - particles.vue@2.7.0
 - particles.vue3@1.7.0
 - tsparticles-editor@1.24.0
 - tsparticles@1.24.0
 - ng-particles-demo@1.9.0
 - editor-demo@1.9.0
 - inferno-particles-demo@1.7.0
 - jquery-particles-demo@1.9.0
 - tsparticles-demo@1.9.0
 - preact-particles-demo@1.9.0
 - react-tsparticles-demo@1.9.0
 - svelte-demo@1.7.0
 - particles.vue-demo@1.9.0
 - particles.vue3-demo@1.7.0
 - tsparticles-preset-60fps@1.9.0
 - tsparticles-preset-background-mask@1.9.0
 - tsparticles-preset-basic@1.9.0
 - tsparticles-preset-big-circles@1.9.0
 - tsparticles-preset-bouncing@1.9.0
 - tsparticles-preset-bubbles@1.9.0
 - tsparticles-preset-fire@1.9.0
 - tsparticles-preset-font-awesome@1.9.0
 - tsparticles-preset-fountain@1.9.0
 - tsparticles-preset-sea-anemone@1.9.0
 - tsparticles-preset-snow@1.9.0
 - tsparticles-preset-stars@1.9.0
 - tsparticles-shape-bubble@1.9.0
 - tsparticles-shape-confetti@1.9.0
 - tsparticles-shape-heart@1.9.0
 - tsparticles-shape-multiline-text@1.9.0
 - tsparticles-shape-rounded-rect@1.9.0
 - tsparticles-shape-spiral@1.9.0
 - cra-template-particles-typescript@1.7.0
 - cra-template-particles@1.7.0
 - tsparticles-website@1.24.0

matteobruni committed:
fix: fixed emitters after 1.24.0

matteobruni committed:
chore(release): published new version

 - ng-particles@2.7.1
 - inferno-particles@1.7.1
 - jquery-particles@1.24.1
 - preact-particles@1.24.1
 - react-tsparticles@1.24.1
 - svelte-particles@1.7.1
 - particles.vue@2.7.1
 - particles.vue3@1.7.1
 - tsparticles-editor@1.24.1
 - tsparticles@1.24.1
 - ng-particles-demo@1.9.1
 - editor-demo@1.9.1
 - inferno-particles-demo@1.7.1
 - jquery-particles-demo@1.9.1
 - tsparticles-demo@1.9.1
 - preact-particles-demo@1.9.1
 - react-tsparticles-demo@1.9.1
 - svelte-demo@1.7.1
 - particles.vue-demo@1.9.1
 - particles.vue3-demo@1.7.1
 - tsparticles-preset-60fps@1.9.1
 - tsparticles-preset-background-mask@1.9.1
 - tsparticles-preset-basic@1.9.1
 - tsparticles-preset-big-circles@1.9.1
 - tsparticles-preset-bouncing@1.9.1
 - tsparticles-preset-bubbles@1.9.1
 - tsparticles-preset-fire@1.9.1
 - tsparticles-preset-font-awesome@1.9.1
 - tsparticles-preset-fountain@1.9.1
 - tsparticles-preset-sea-anemone@1.9.1
 - tsparticles-preset-snow@1.9.1
 - tsparticles-preset-stars@1.9.1
 - tsparticles-shape-bubble@1.9.1
 - tsparticles-shape-confetti@1.9.1
 - tsparticles-shape-heart@1.9.1
 - tsparticles-shape-multiline-text@1.9.1
 - tsparticles-shape-rounded-rect@1.9.1
 - tsparticles-shape-spiral@1.9.1
 - cra-template-particles-typescript@1.7.1
 - cra-template-particles@1.7.1

matteobruni committed:
Merge pull request #1407 from matteobruni/v1

1.24.1

matteobruni committed:
build: website updated

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
Merge pull request #1411 from matteobruni/v1

build: website updated

matteobruni committed:
fix: removed an old console.dir, fixes #1412

matteobruni committed:
Merge branch 'v1' of https://github.com/matteobruni/tsparticles into v1

matteobruni committed:
chore(release): published new version

 - react-tsparticles@1.24.2
 - react-tsparticles-demo@1.9.2
 - cra-template-particles-typescript@1.7.2
 - cra-template-particles@1.7.2
 - tsparticles-website@1.24.1

matteobruni committed:
Merge branch 'main' into v1

matteobruni committed:
Merge pull request #1413 from matteobruni/v1

React 1.24.2

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: updated confetti

dependabot[bot] committed:
build(deps-dev): bump @vue/cli-plugin-babel from 4.5.11 to 4.5.12

Bumps [@vue/cli-plugin-babel](https://github.com/vuejs/vue-cli/tree/HEAD/packages/@vue/cli-plugin-babel) from 4.5.11 to 4.5.12.
- [Release notes](https://github.com/vuejs/vue-cli/releases)
- [Changelog](https://github.com/vuejs/vue-cli/blob/dev/CHANGELOG.md)
- [Commits](https://github.com/vuejs/vue-cli/commits/v4.5.12/packages/@vue/cli-plugin-babel)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump terser-webpack-plugin from 5.0.3 to 5.1.1

Bumps [terser-webpack-plugin](https://github.com/webpack-contrib/terser-webpack-plugin) from 5.0.3 to 5.1.1.
- [Release notes](https://github.com/webpack-contrib/terser-webpack-plugin/releases)
- [Changelog](https://github.com/webpack-contrib/terser-webpack-plugin/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/terser-webpack-plugin/compare/v5.0.3...v5.1.1)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1416 from matteobruni/dependabot/npm_and_yarn/dev/terser-webpack-plugin-5.1.1

build(deps-dev): bump terser-webpack-plugin from 5.0.3 to 5.1.1

matteobruni committed:
Merge pull request #1415 from matteobruni/dependabot/npm_and_yarn/dev/vue/cli-plugin-babel-4.5.12

build(deps-dev): bump @vue/cli-plugin-babel from 4.5.11 to 4.5.12

dependabot[bot] committed:
build(deps-dev): bump @types/react-dom from 17.0.0 to 17.0.3

Bumps [@types/react-dom](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/react-dom) from 17.0.0 to 17.0.3.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/react-dom)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump inferno from 7.4.7 to 7.4.8

Bumps [inferno](https://github.com/infernojs/inferno) from 7.4.7 to 7.4.8.
- [Release notes](https://github.com/infernojs/inferno/releases)
- [Commits](https://github.com/infernojs/inferno/compare/v7.4.7...v7.4.8)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @vue/cli-plugin-eslint from 4.5.11 to 4.5.12

Bumps [@vue/cli-plugin-eslint](https://github.com/vuejs/vue-cli/tree/HEAD/packages/@vue/cli-plugin-eslint) from 4.5.11 to 4.5.12.
- [Release notes](https://github.com/vuejs/vue-cli/releases)
- [Changelog](https://github.com/vuejs/vue-cli/blob/dev/CHANGELOG.md)
- [Commits](https://github.com/vuejs/vue-cli/commits/v4.5.12/packages/@vue/cli-plugin-eslint)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump rollup from 2.38.5 to 2.42.4

Bumps [rollup](https://github.com/rollup/rollup) from 2.38.5 to 2.42.4.
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v2.38.5...v2.42.4)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1417 from matteobruni/dependabot/npm_and_yarn/dev/types/react-dom-17.0.3

build(deps-dev): bump @types/react-dom from 17.0.0 to 17.0.3

matteobruni committed:
Merge pull request #1418 from matteobruni/dependabot/npm_and_yarn/dev/inferno-7.4.8

build(deps): bump inferno from 7.4.7 to 7.4.8

matteobruni committed:
Merge pull request #1419 from matteobruni/dependabot/npm_and_yarn/dev/vue/cli-plugin-eslint-4.5.12

build(deps-dev): bump @vue/cli-plugin-eslint from 4.5.11 to 4.5.12

matteobruni committed:
Merge pull request #1420 from matteobruni/dependabot/npm_and_yarn/dev/rollup-2.42.4

build(deps-dev): bump rollup from 2.38.5 to 2.42.4

matteobruni committed:
build: aligned to 1.24.1

matteobruni committed:
feat: added mass to particle, this will improve collisions

matteobruni committed:
build: improved some imports

matteobruni committed:
build: added docs to main demo

matteobruni committed:
build: fixed samples url

matteobruni committed:
chore(release): published new version

 - ng-particles@2.7.2
 - inferno-particles@1.7.2
 - jquery-particles@1.24.2
 - preact-particles@1.24.2
 - react-tsparticles@1.24.3
 - svelte-particles@1.7.2
 - particles.vue@2.7.2
 - particles.vue3@1.7.2
 - tsparticles-editor@1.24.2
 - tsparticles@1.24.2
 - ng-particles-demo@1.9.2
 - editor-demo@1.9.2
 - inferno-particles-demo@1.7.2
 - jquery-particles-demo@1.9.2
 - tsparticles-demo@1.9.2
 - preact-particles-demo@1.9.2
 - react-tsparticles-demo@1.9.3
 - svelte-demo@1.7.2
 - particles.vue-demo@1.9.2
 - particles.vue3-demo@1.7.2
 - tsparticles-preset-60fps@1.9.2
 - tsparticles-preset-background-mask@1.9.2
 - tsparticles-preset-basic@1.9.2
 - tsparticles-preset-big-circles@1.9.2
 - tsparticles-preset-bouncing@1.9.2
 - tsparticles-preset-bubbles@1.9.2
 - tsparticles-preset-fire@1.9.2
 - tsparticles-preset-font-awesome@1.9.2
 - tsparticles-preset-fountain@1.9.2
 - tsparticles-preset-sea-anemone@1.9.2
 - tsparticles-preset-snow@1.9.2
 - tsparticles-preset-stars@1.9.2
 - tsparticles-shape-bubble@1.9.2
 - tsparticles-shape-confetti@1.9.2
 - tsparticles-shape-heart@1.9.2
 - tsparticles-shape-multiline-text@1.9.2
 - tsparticles-shape-rounded-rect@1.9.2
 - tsparticles-shape-spiral@1.9.2
 - cra-template-particles-typescript@1.7.3
 - cra-template-particles@1.7.3
 - tsparticles-website@1.24.2

matteobruni committed:
build: updated website

matteobruni committed:
Merge pull request #1421 from matteobruni/v1

1.24.2

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: merge done with 1.24.2

matteobruni committed:
build: refactoring particle class

dependabot[bot] committed:
build(deps): bump react-dom from 17.0.1 to 17.0.2

Bumps [react-dom](https://github.com/facebook/react/tree/HEAD/packages/react-dom) from 17.0.1 to 17.0.2.
- [Release notes](https://github.com/facebook/react/releases)
- [Changelog](https://github.com/facebook/react/blob/master/CHANGELOG.md)
- [Commits](https://github.com/facebook/react/commits/v17.0.2/packages/react-dom)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump webpack-cli from 4.2.0 to 4.6.0

Bumps [webpack-cli](https://github.com/webpack/webpack-cli) from 4.2.0 to 4.6.0.
- [Release notes](https://github.com/webpack/webpack-cli/releases)
- [Changelog](https://github.com/webpack/webpack-cli/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack/webpack-cli/compare/webpack-cli@4.2.0...webpack-cli@4.6.0)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1423 from matteobruni/dependabot/npm_and_yarn/dev/react-dom-17.0.2

build(deps): bump react-dom from 17.0.1 to 17.0.2

dependabot[bot] committed:
build(deps): bump rxjs from 6.6.6 to 6.6.7

Bumps [rxjs](https://github.com/reactivex/rxjs) from 6.6.6 to 6.6.7.
- [Release notes](https://github.com/reactivex/rxjs/releases)
- [Changelog](https://github.com/ReactiveX/rxjs/blob/6.6.7/CHANGELOG.md)
- [Commits](https://github.com/reactivex/rxjs/compare/6.6.6...6.6.7)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @vue/cli-service from 4.5.11 to 4.5.12

Bumps [@vue/cli-service](https://github.com/vuejs/vue-cli/tree/HEAD/packages/@vue/cli-service) from 4.5.11 to 4.5.12.
- [Release notes](https://github.com/vuejs/vue-cli/releases)
- [Changelog](https://github.com/vuejs/vue-cli/blob/dev/CHANGELOG.md)
- [Commits](https://github.com/vuejs/vue-cli/commits/v4.5.12/packages/@vue/cli-service)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1424 from matteobruni/dependabot/npm_and_yarn/dev/webpack-cli-4.6.0

build(deps-dev): bump webpack-cli from 4.2.0 to 4.6.0

matteobruni committed:
Merge pull request #1425 from matteobruni/dependabot/npm_and_yarn/dev/rxjs-6.6.7

build(deps): bump rxjs from 6.6.6 to 6.6.7

matteobruni committed:
Merge pull request #1426 from matteobruni/dependabot/npm_and_yarn/dev/vue/cli-service-4.5.12

build(deps-dev): bump @vue/cli-service from 4.5.11 to 4.5.12

dependabot[bot] committed:
build(deps-dev): bump postcss from 8.2.7 to 8.2.8

Bumps [postcss](https://github.com/postcss/postcss) from 8.2.7 to 8.2.8.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/main/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/8.2.7...8.2.8)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump svelte from 3.35.0 to 3.36.0

Bumps [svelte](https://github.com/sveltejs/svelte) from 3.35.0 to 3.36.0.
- [Release notes](https://github.com/sveltejs/svelte/releases)
- [Changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte/compare/v3.35.0...v3.36.0)

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @vue/cli-plugin-typescript from 4.5.11 to 4.5.12

Bumps [@vue/cli-plugin-typescript](https://github.com/vuejs/vue-cli/tree/HEAD/packages/@vue/cli-plugin-typescript) from 4.5.11 to 4.5.12.
- [Release notes](https://github.com/vuejs/vue-cli/releases)
- [Changelog](https://github.com/vuejs/vue-cli/blob/dev/CHANGELOG.md)
- [Commits](https://github.com/vuejs/vue-cli/commits/v4.5.12/packages/@vue/cli-plugin-typescript)

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1428 from matteobruni/dependabot/npm_and_yarn/dev/postcss-8.2.8

build(deps-dev): bump postcss from 8.2.7 to 8.2.8

matteobruni committed:
Merge pull request #1430 from matteobruni/dependabot/npm_and_yarn/dev/vue/cli-plugin-typescript-4.5.12

build(deps-dev): bump @vue/cli-plugin-typescript from 4.5.11 to 4.5.12

matteobruni committed:
Merge pull request #1429 from matteobruni/dependabot/npm_and_yarn/dev/svelte-3.36.0

build(deps): bump svelte from 3.35.0 to 3.36.0

matteobruni committed:
build: prepared bundles for bundles packages

matteobruni committed:
chore(release): published new version

 - tsparticles@2.0.0-alpha.6
 - tsparticles-slim@2.0.0-alpha.6
 - ng-particles@3.0.0-alpha.6
 - inferno-particles@2.0.0-alpha.6
 - jquery-particles@2.0.0-alpha.6
 - preact-particles@2.0.0-alpha.6
 - react-tsparticles@2.0.0-alpha.6
 - svelte-particles@2.0.0-alpha.6
 - particles.vue@3.0.0-alpha.6
 - particles.vue3@2.0.0-alpha.6
 - ng-particles-demo@2.0.0-alpha.6
 - editor-demo@2.0.0-alpha.6
 - inferno-particles-demo@2.0.0-alpha.6
 - jquery-particles-demo@2.0.0-alpha.6
 - preact-particles-demo@2.0.0-alpha.6
 - react-tsparticles-demo@2.0.0-alpha.6
 - svelte-demo@2.0.0-alpha.6
 - tsparticles-demo@2.0.0-alpha.6
 - particles.vue-demo@2.0.0-alpha.6
 - particles.vue3-demo@2.0.0-alpha.6
 - tsparticles-editor@2.0.0-alpha.6
 - tsparticles-engine@2.0.0-alpha.6
 - tsparticles-interaction-light@2.0.0-alpha.6
 - tsparticles-interaction-external-attract@2.0.0-alpha.6
 - tsparticles-interaction-external-bounce@2.0.0-alpha.6
 - tsparticles-interaction-external-bubble@2.0.0-alpha.6
 - tsparticles-interaction-external-connect@2.0.0-alpha.6
 - tsparticles-interaction-external-grab@2.0.0-alpha.6
 - tsparticles-interaction-external-repulse@2.0.0-alpha.6
 - tsparticles-interaction-external-trail@2.0.0-alpha.6
 - tsparticles-interaction-particles-attract@2.0.0-alpha.6
 - tsparticles-interaction-particles-collisions@2.0.0-alpha.6
 - tsparticles-interaction-particles-links@2.0.0-alpha.6
 - tsparticles-interaction-particles-move@2.0.0-alpha.6
 - tsparticles-interaction-particles-parallax@2.0.0-alpha.6
 - tsparticles-interaction-particles-repulse@2.0.0-alpha.6
 - tsparticles-plugin-absorbers@2.0.0-alpha.6
 - tsparticles-plugin-emitters@2.0.0-alpha.6
 - tsparticles-plugin-infection@2.0.0-alpha.6
 - tsparticles-plugin-polygon-mask@2.0.0-alpha.6
 - tsparticles-preset-big-circles@2.0.0-alpha.6
 - tsparticles-preset-bubbles@2.0.0-alpha.6
 - tsparticles-preset-fire@2.0.0-alpha.6
 - tsparticles-preset-fountain@2.0.0-alpha.6
 - tsparticles-preset-links@2.0.0-alpha.6
 - tsparticles-preset-sea-anemone@2.0.0-alpha.6
 - tsparticles-preset-snow@2.0.0-alpha.6
 - tsparticles-preset-stars@2.0.0-alpha.6
 - tsparticles-shape-bubble@2.0.0-alpha.6
 - tsparticles-shape-circle@2.0.0-alpha.6
 - tsparticles-shape-confetti@2.0.0-alpha.6
 - tsparticles-shape-heart@2.0.0-alpha.6
 - tsparticles-shape-image@2.0.0-alpha.6
 - tsparticles-shape-line@2.0.0-alpha.6
 - tsparticles-shape-multiline-text@2.0.0-alpha.6
 - tsparticles-shape-polygon@2.0.0-alpha.6
 - tsparticles-shape-rounded-rect@2.0.0-alpha.6
 - tsparticles-shape-spiral@2.0.0-alpha.6
 - tsparticles-shape-square@2.0.0-alpha.6
 - tsparticles-shape-star@2.0.0-alpha.6
 - tsparticles-shape-text@2.0.0-alpha.6
 - cra-template-particles-typescript@2.0.0-alpha.6
 - cra-template-particles@2.0.0-alpha.6
 - tsparticles-updater-angle@2.0.0-alpha.6
 - tsparticles-updater-color@2.0.0-alpha.6
 - tsparticles-updater-life@2.0.0-alpha.6
 - tsparticles-updater-opacity@2.0.0-alpha.6
 - tsparticles-updater-orbit@2.0.0-alpha.6
 - tsparticles-updater-out-modes@2.0.0-alpha.6
 - tsparticles-updater-size@2.0.0-alpha.6
 - tsparticles-updater-stroke-color@2.0.0-alpha.6
 - tsparticles-website@2.0.0-alpha.5

matteobruni committed:
build: config updates
